### ISSUE TITLE
[8.1.0] skipColumnOnPaste doesn't work if hiddenColumns and columnSorting are enabled

### ISSUE THREAD

**Issue by mrpiotr-dev** [handsontable/handsontable#7355]
### Description
#7299 introduces a regression with `skipColumnOnPaste` option. It allows users to paste data into cells marked to be skipped in pasting.

During work on #7171, I noticed that the following test spec doesn't succeed with a fresh build: https://github.com/handsontable/handsontable/commit/b1b4b90ff3dc5284496352c8033e0a72c691f429

### Steps to reproduce
1. Initialize Handsontable with the following configuration:
```
data: Handsontable.helper.createSpreadsheetData(1, 2),
columns: [
  { skipColumnOnPaste: true },
  {}
],
hiddenColumns: true,
columnSorting: true,
```
2. Select ` A1 ` cell.
3. Paste `this should be in B1` into the selected cell.

### Demo
https://codesandbox.io/s/handsontableexamples-pull-request-forked-ruqn9?file=/src/index.js

### Your environment
* Handsontable version: Current develop, post 8.1.0 changes

**Comment by swistach** [comment]
Due to problems with overwriting cell meta, I have to move this issue to "On hold"

**Comment by aninde** [comment]
I am closing this issue as fixed in the latest v12.1.2 https://jsfiddle.net/m4j7d3zk/, fixed from LTS 8.1. https://jsfiddle.net/v5czm6py/1/.
